src/esp32/motor.rs

```diff
@@ -265,7 +285,12 @@ where
     PWM: PwmPin<Duty = u32>,
 {
     pub fn new(dir: DIR, pwm: PWM, dir_flip: bool) -> Self {
-        Self { dir, pwm, dir_flip }
+        Self {
+            dir,
+            pwm,
+            dir_flip,
+            max_rpm: f64::default(),
```

Comment:
what is the value of a f64 default? can you add a comment so it's easy to figure out?